### ISSUE TITLE
Expansion of Ohio state abbreviation getting confused with numeric expression

### ISSUE THREAD

**Issue by bsheneman** [openvenues/libpostal#271]
The address expander on both command line and in the Python library often fails on addresses that have the state Ohio abbreviated (as "oh" or "OH") -- it replaces the state abbreviation with the number '0'.

#### Example 1 (fails)

```
./libpostal "4998 Vanderbilt Dr, Columbus, OH 43213, USA"

4998 vanderbilt drive columbus 0 43213 usa
4998 vanderbilt doctor columbus 0 43213 usa
```

#### Example 2 (fails)

```
./libpostal "1556 LOGAN AVE NW, CANTON, OH 44703"

1556 logan avenue northwest canton 0 44703
1556 logan avenue nw canton 0 44703
```

#### Example 3 (works)

```
./libpostal "4998 Vanderbilt Dr, Columbus, Ohio 43213, USA"

4998 vanderbilt drive columbus ohio 43213 usa
4998 vanderbilt doctor columbus ohio 43213 usa
```

I am using pypostal 1.0, and libpostal 1.0.0 for these tests.

**Comment by albarrentine** [comment]
Hm, that's a bug in that numeric expressions (think "twenty-one oh one") being expanded before the abbreviated phrases are considered. Really "oh" isn't something that should be expanded unless the other phrases to its left or right are also numeric phrases. Will try to fix shortly.

**Comment by albarrentine** [comment]
Fixed with #274
